Workbooks/HDInsight/At-Scale/HDInsightAtScale.workbook

```diff
@@ -1851,7 +1943,7 @@
                               },
                               "customWidth": "50",
                               "showPin": true,
-                              "name": "query - 0 - Copy - Copya"
+                              "name": "query - 0 - Copy - Copy"
```

Comment:
For localization reasons, we need to have these names unique, can you go over these and give each of these sections/items a unique name? Would also be good to change to a description of what the section is, in case a customer inspects certain sections of the workbook and it'll show the description instead of 'query - 0 - Copy - Copy'

Reply:
Removed these